verifier/pom.xml

```diff
@@ -206,20 +206,45 @@
       <version>${syndesis.version}</version>
     </dependency>
 
+    <dependency>
+      <groupId>io.syndesis</groupId>
+      <artifactId>sql-common</artifactId>
+      <version>1.2-SNAPSHOT</version>
+    </dependency>
+    
+    <dependency>
+      <groupId>io.syndesis</groupId>
+      <artifactId>sql-connector</artifactId>
+      <version>1.2-SNAPSHOT</version>
+    </dependency>
+
+    <dependency>
+      <groupId>io.syndesis</groupId>
+      <artifactId>sql-start-connector</artifactId>
+      <version>1.2-SNAPSHOT</version>
+    </dependency>
+
     <dependency>
       <groupId>io.syndesis</groupId>
       <artifactId>sql-stored-connector</artifactId>
-      <version>${syndesis.version}</version>
+      <version>1.2-SNAPSHOT</version>
```

Comment:
How to deal with depending on a SNAPSHOTS? How does this work in the mono repo? Hardcoding for now..